servatrice/servatrice.ini.example

```diff
@@ -107,6 +107,14 @@ allowedpunctuation=_.-
 ; If a username can begin with punctuation defined in allowedpunctuation
 allowpunctuationprefix=false
 
+; Disallow usernames containing these words. This list is comma seperated
+disallowedwords="admin"
+
+; Disallow usernames matching these regular expressions. This list is comma
+; separated, hence you cannot use commas in your expressions. Backslashes must
+; be escaped, so `\w+\d+` becomes `\\w+\\d+`
+disallowedregexp=""
```

Comment:
I'd keep it, just adding a warning that adding a lot of regular expressions or some particularly complex regexp could lead to resource exhaustion on the server.